app/controllers/foods_controller.rb

```diff
@@ -1,6 +1,20 @@
 class FoodsController < ApplicationController
   before_action :load_food, only: :show
 
+  def index
+    @foods = Food.search_by_name(params[:name])
+
+    if @foods.blank?
+      @foods = Food
+      flash.now[:danger] = t "search_blank"
+    end
+
+    @foods = @foods.enabled.recent_foods
```

Comment:
```suggestion
    @foods = Food.search_by_name(params[:name]).enabled.recent_foods
```
Viết thế này có bỏ đc đoạn check blank kia ko em

Reply:
em có dòng ở trên để search trước r ạ , searh nếu trả về bank thì nó sẽ in ra không tìm thấy , còn nếu không nó sẽ in ra các cái foods trong danh sách foods tìm được mà status của nó là enabled

Reply:
Sao em không kiểm tra ngoài view á 
if @foods.any?
  do some thing
else 
  in thông báo
end